examples/demo10.py

```diff
@@ -201,10 +217,11 @@ def main(argv):
         # orientations stepped uniformly in angle, making a ring in e1-e2 space.
         # We're drawing each profile at 20 different orientations and then skipping to the
         # next galaxy in the list.  So theta steps by 1/20 * 360 degrees:
-        theta = k/20. * 360. * galsim.degrees
+        theta_deg = (k%20) * 360. / 20
+        theta = theta_deg * galsim.degrees
 
         # The index needs to increment every 20 objects so we use k/20 using integer math.
-        index = k / 20
+        index = k // 20
```

Comment:
This was a bit of pre-planning for making GalSim compatible with [Python 3](https://github.com/GalSim-developers/GalSim/issues/534).  The `//` operator has the same meaning in both Python 2 and 3.  But `/` does not.

Ideally, I'd like to never use `from __future__ import division` if possible, since I don't actually like having integer division return a float as it does in Python 3.  That runs against all my instincts as a C/C++ programmer.  `5 / 3` is 1, not 1.66666667.   But writing `//` at least makes me more cognizant of the issue without any ambiguity.